packages/api/src/apps/api/app.ts

```diff
@@ -0,0 +1,65 @@
+import assert from "assert";
```

Comment:
It seems like we have 4 levels of nesting in this package folder structure, and the files seem to be a bit sparse. Curious why we need to specify api twice in the path -- that seems a bit redundant. In terms of moving through the repo, I think going for fewer levels of folder nesting is ideal.

One rule of thumb is that you should probably have around 10 items per level of nesting before you need to create a new level. If you have way less than that at a level, for example, then that should be the bottom level and there should be no folders below. If we need to restructure later to encompass more complexity or files, that seems preferable to starting with a complex, but sparse dir structure.

Thoughts?

Thoughts?

Reply:
yea i could flatten the app folder with just the api app for now. I forsee though splitting this app into potentially 2 other apps, if not more. You can imagine an app which is ingesting price feed data, smart contract data, maybe data from Contentful or something else and putting them in database. Another express app could be running serverless which handles requests. These would fill up the app folder and each have their own readme.